Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Search/SearchWidget.cpp

```diff
@@ -175,10 +175,21 @@ namespace AzToolsFramework
                 SetTypeFilters(buildTypesFilterList());
             }
 
-            AZ::IO::Path projectPath{ AZ::Utils::GetProjectPath() };
-            AZ::IO::Path enginePath{ AZ::Utils::GetEnginePath() };
-            auto engineFilter = new EngineFilter();
-            engineFilter->SetEngineAndProject(enginePath, projectPath);
+            const AZ::IO::Path enginePath{ AZ::Utils::GetEnginePath() };
+            AZ::IO::Path engineAssets = enginePath;
+            engineAssets = engineAssets.Append("Assets/Engine").LexicallyNormal();
+            AZ::IO::Path setregAssets = enginePath;
+            setregAssets = setregAssets.Append("Registry").LexicallyNormal();
```

Comment:
Nit: This can be condensed a bit with the `/` operators that you can use with `AZ::IO::Path`
```suggestion
            AZ::IO::Path engineAssets = (enginePath / "Assets" / "Engine").LexicallyNormal();
            AZ::IO::Path setregAssets = (enginePath / "Registry").LexicallyNormal();
```

Reply:
Plus the paths don't need to be normalized.

AZ::IO::Path members functions can deal with paths that aren't normalized, normalization is for when the path is used outside of the AZ::IO::Path API.

So using the `IsRelativeTo` API below doesn't require normalization

Reply:
Sounds good, done!